src/package.json

```diff
@@ -15,7 +15,7 @@
   "license": "MIT",
   "author": "Nicholas Lee <nicholaslee119@gmail.com>",
   "main": "dist/CheckBox",
-  "types": "typings/index.d.ts",
+  "types": "dist/index.d.ts",
```

Comment:
There is no `dist/index.d.ts` after building
Actually, I have tried to generate the dist/index.d.ts for iOS and Android both.
I didn't find a good way to do that, if you knew how to generate `dist/index.d.ts` for iOS and Android both, please tell us